style-sheets/primevue-components-overrides.scss

```diff
@@ -1140,4 +1140,133 @@ label {
+.#{variables.$primevue-prefix}-tabview-header
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    display: flex;
+    justify-content: space-between;
+    min-width: 11.16rem;
+    font-weight: 400;
+    background-color: map.get(lightTheme.$light-theme, 'border-subtle-00');
+    border-color: map.get(lightTheme.$light-theme, 'border-subtle-00');
+    padding-bottom: 0.6875rem;
+    border-radius: 0;
+    border-bottom: none;
+    border-right: none;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav-link
+.#{variables.$primevue-prefix}-tabview-title {
+    font-size: 0.875rem;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:first-child
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-top-left-radius: 0.25rem;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:nth-last-child(2)
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-top-right-radius: 0.25rem;
+    border-top-left-radius: 0;
+    border-right: solid 0.0625rem map.get(lightTheme.$light-theme, 'border-subtle-00');
+    border-left: none;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header.#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    font-weight: 700;
+    border-top: solid 0.125rem map.get(
+        lightTheme.$light-theme,
+        'border-interactive'
+    ) !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:last-child.#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-left: 0.0625rem solid map.get(lightTheme.$light-theme, 'text-secondary') !important;
+    border-right: 0.0625rem solid map.get(lightTheme.$light-theme, 'text-secondary') !important;
+    background-color: blue($color: #000000) !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:not(:first-child):not(:nth-last-child(2)).#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link{
+    border-right: 0.0625rem solid map.get(lightTheme.$light-theme, 'border-strong-01') !important;
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:not(:first-child).#{variables.$primevue-prefix}-highlight
+.#{variables.$primevue-prefix}-tabview-nav-link {
+    border-left: solid 0.0625rem map.get(lightTheme.$light-theme, 'border-strong-01');
+}
+
+.#{variables.$primevue-prefix}-tabview
+.#{variables.$primevue-prefix}-tabview-nav
+.#{variables.$primevue-prefix}-tabview-header:not(.#{variables.$primevue-prefix}-highlight)
```

Comment:
and this one, there are so many cases, I'm not 100% sure.

Reply:
this select the hover that it is not active